### PR TITLE
many: change name and input of CheckProvenance to clarify usage

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

File: asserts/snapasserts/snapasserts.go [hunk 1]
```diff
@@ -145,13 +150,13 @@ func CrossCheckProvenance(instanceName string, snapRev *asserts.SnapRevision, sn
...
+// CheckProvenanceWithVerifiedRevision checks that the given snap has
+// the same provenance as of the provided snap-revision.
 // It is intended to be called safely on snaps for which a matching
-// and authorized snap-revision has been already found which specify
-// the given provenance.
+// and authorized snap-revision has been already found and cross-checked.
 // Its purpose is to check that a blob has not been re-signed under an
 // inappropriate provenance.
-func CheckProvenance(snapPath, provenance string) error {
+func CheckProvenanceWithVerifiedRevision(snapPath string, verifiedRev *asserts.SnapRevision) error {
```
Comment (mvo5) [inline asserts/snapasserts/snapasserts.go:159]: I like this change

**Review by mvo5 — APPROVED** [review]
Thank you

**Review by MiguelPires — APPROVED** [review]
LGTM

**Review by mardy — APPROVED** [review]
LGTM, thanks!

**Pull request merged**